digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleSpeedLimitDao.scala

```diff
@@ -0,0 +1,517 @@
+package fi.liikennevirasto.digiroad2.dao.linearasset
+
+import fi.liikennevirasto.digiroad2._
+import fi.liikennevirasto.digiroad2.asset._
+import fi.liikennevirasto.digiroad2.linearasset._
+import fi.liikennevirasto.digiroad2.oracle.MassQuery
+import org.joda.time.DateTime
+import slick.driver.JdbcDriver.backend.Database
+import Database.dynamicSession
+import _root_.oracle.sql.STRUCT
+import com.github.tototoshi.slick.MySQLJodaSupport._
+import fi.liikennevirasto.digiroad2.client.vvh.VVHClient
+import fi.liikennevirasto.digiroad2.dao.Queries.bytesToPoint
+import fi.liikennevirasto.digiroad2.dao.{Queries, Sequences}
+import fi.liikennevirasto.digiroad2.service.RoadLinkService
+import fi.liikennevirasto.digiroad2.service.linearasset.Measures
+import org.slf4j.LoggerFactory
+import slick.jdbc.StaticQuery.interpolation
+import slick.jdbc.{GetResult, PositionedParameters, PositionedResult, SetParameter, StaticQuery => Q}
+
+case class PersistedSpeedLimit(id: Long, linkId: Long, sideCode: SideCode, value: Option[Int], startMeasure: Double, endMeasure: Double,
```

Comment:
PersistedSpeedLimit this class could be move to SpeedLimit.scala like it was done with linear (LinearAsset.scala)